### DISCUSSION TITLE
Add my own OSINT tool to the dropdown list

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#2707] [Ideas]
Still relatively new to SO, so forgive the newb-ness of this question...

When you left click and get the dropdown with Include, Exclude, Actions... I know this is a lot to hope for, but is that list configurable? I have my own OSINT tool that I use to lookup domains and IPs. I would like to call the URL of my tool and pass it, for instance, https://recon.cyberiq.ai/Network/?ip=[IP ADDRESS], which would call my app and pass it the IP instead of selecting VirusTotals or Google.

Is there a configuration file in SO that I could add an item to that list?

**Comment by cm-ops** [comment]
If you want to add your own custom actions see https://docs.securityonion.net/en/latest/alerts.html?highlight=Actions#actions

**Reply by ghost** [reply]
Awesome! Thanks a lot. And thanks for replying so quickly.

**Reply by ghost** [reply]
Also, I had to do this with the hunt.actions.json file so the choice would show up in Hunt. Everything is working! Thank you.
